include/deal.II/lac/read_write_vector.templates.h

```diff
@@ -164,11 +165,12 @@ namespace LinearAlgebra
 #ifdef DEAL_II_WITH_TRILINOS
   template <typename Number>
   ReadWriteVector<Number> &
-  ReadWriteVector<Number>::operator = (const TrilinosWrappers::MPI::Vector &trilinos_vec)
+  ReadWriteVector<Number>::import(const Epetra_MultiVector &multivector,
```

Comment:
If it's just an internal helper function I guess it's fine. Thanks for the clarification.